web/src/components/metrics/metrics-content.tsx

```diff
@@ -12,20 +12,21 @@ import {
   ChartThemeColor
 } from '@patternfly/react-charts';
 import * as React from 'react';
+import { useTranslation } from 'react-i18next';
 import { NamedMetric } from '../../api/loki';
 import { MetricType } from '../../model/flow-query';
+import { LOCAL_STORAGE_OVERVIEW_METRICS_DIMENSION_KEY, useLocalStorage } from '../../utils/local-storage-hook';
```

Comment:
Side remark: We might want to use some formatting tool to organize import automatically like we have on the go side
E.g. https://medium.com/weekly-webtips/how-to-sort-imports-like-a-pro-in-typescript-4ee8afd7258a

Reply:
Added to https://issues.redhat.com/browse/NETOBSERV-268